source/assets/scripts/main.js

```diff
@@ -204,6 +207,8 @@ function addCollapsibleControls() {
 /**
  * Add event handler to all add buttons of each day's list. When user click add btn,
  * a new task div will appear
+ * 
+ * @param {object}task a task object with its corresponding properties
```

Comment:
Add space after {object} & capitalization 
`{object} task A task object with its corresponding properties `

Reply:
And can you clarify why we need this parameter? It seems like we never use it with parameter through out main.js.